### PR TITLE
TW-555 Fix displaying icons of some tokens

### PR DESCRIPTION
**Pull request by keshan3262**

### CODE REVIEW

File: src/components/token-icon/token-icon.tsx [hunk 1]
```diff
@@ -38,10 +42,29 @@ type TokenIconImageProps = Props & {
...
   const colors = useColors();
+  const dispatch = useDispatch();
+  const isKnownIpfsSvg = useIsKnownIpfsSvgSelector(thumbnailUri ?? '');
+  const [triedSvg, setTriedSvg] = useState(false);
 
   const { metadata } = useNetworkInfo();
 
+  const handleLoadableTokenIconError = useCallback(() => {
+    if (isFromIpfs && !triedSvg) {
+      dispatch(addKnownIpfsSvg(thumbnailUri!));
```
Comment (lendihop) [inline src/components/token-icon/token-icon.tsx:55]: pls fix github warnings

**Review by lendihop — APPROVED** [review]

**Review by Boltman92 — DISMISSED** [review]

**Review by Boltman92 — APPROVED** [review]

**Review by lendihop — DISMISSED** [review]

**Review by IrynaKhyzhynska — DISMISSED** [review]
android done build number: 1.15_711

**Review by vladvn1905 — DISMISSED** [review]
ios done build: 1.15_711

**Review by Boltman92 — APPROVED** [review]

**Review by lendihop — APPROVED** [review]

**Review by vladvn1905 — APPROVED** [review]
ios done build: 1.15_724

**Pull request merged**